ts/components/ui/FooterButtons.tsx

```diff
@@ -0,0 +1,53 @@
+import { Button, NativeBase, Text, View } from "native-base";
+import { connectStyle } from "native-base-shoutem-theme";
+import mapPropsToStyleNames from "native-base/src/utils/mapPropsToStyleNames";
+import * as React from "react";
+
+type OwnProps = {
+  title: string;
+};
+
+type Props = Readonly<{
+  leftButton: NativeBase.Button & OwnProps;
```

Comment:
Because I just want to give the opportunity to customize button without compromising the structure (Button + Text)

We can consider to pass Button components if we need other stuffs inside button, i.e. an icon.

Reply:
What about
```typescript
type Props = Readonly<{
   leftButton: NativeBase.Button & OwnProps;
   rightButton: NativeBase.Button & OwnProps; 
}> &
(
    Readonly<{ inlineHalf: true; inlineOneThird: false; }>
  | Readonly<{ inlineHalf: false; inlineOneThird: true; }>
);
```
?